guidance/credential-sharing.md

```diff
@@ -1,27 +1,21 @@
 # Helpful Engineering Shared Credentials
 
-Whenever possible, no account should be shared with other users.  However, in some instances (e.g. social media accounts, some email accounts, *et cetera*) it will be required to share a single set of credentials with multiple users using a tool called [LastPass](https://www.lastpass.com).
+Whenever possible, no account should be shared with other users.  However, in some instances (e.g. social media accounts, some email accounts, *et cetera*) it will be required to share a single set of credentials with multiple users by using a tool called **[1Password](https://1password.com)**.
 
- ⚠️ Credentials should **never** be shared outside LastPass: please **don't** send passwords in plaintext by any other means.
+The DevOps and Infosec teams maintain a master account where all credentials are stored and shared with the approved users. Passwords are being rotated regularly for security reasons, and the new ones will automatically synchronize in the other users' 1Password vaults. 
```

Comment:
Currently it looks like DevOps and Infosec are nearly like one big team; I would dare to say we're nearly DevSecOps (in the order you prefer), though @dmangot has very concrete opinions about this kind of naming conventions.

Reply:
I think the names of the teams are fine, but I don't think that deleting the old text in favor of the new one sufficiently conveys the importance of not sharing the passwords by other means.  You're leaving as a mental exercise for the reader to come to that conclusion which seems obvious to you with your background, but may not be obvious to them.

Reply:
I haven't deleted the text in red, it has just moved.

Reply:
May we mark this conversation as solved?